gateway/testutil.go

```diff
@@ -2031,3 +2033,69 @@ func (m *testMessageAdapter) Channel() (string, error) {
 func (m *testMessageAdapter) Payload() (string, error) {
 	return m.Msg, nil
 }
+
+func TestHelperSSEServer(tb testing.TB) *httptest.Server {
+	tb.Helper()
+	return httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
```

Comment:
```suggestion
	return httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, _ *http.Request) {
```

Reply:
why doesn't golangcilint catch this in pre-commit?

Reply:
need to check lint rules in that case.